src/_webpack/search.js

```diff
@@ -1,3 +1,6 @@
+const elasticlunr = require('elasticlunr')
+const lunr = require('elasticlunr');
```

Comment:
Haven't looked at this properly (exciting changes though!), but if this is otherwise ready to go, would be good to remove the semicolon here to keep it Standard

Reply:
Is it possible to configure Standard to prefer semicolons? I'm the type who likes semicolons in my JavaScript (and believe it's the default with ESLint), but maybe I need to get with the times...

Reply:
Hah, [apparently not](https://standardjs.com/#i-disagree-with-rule-x-can-you-change-it)

Reply:
Love the pro tip! This semicolon is actually required because of the iife and passes the checks. An option would be to rewrite this as a module

Reply:
I think I'll merge this with the semicolon for now - but happy to pick up rewriting to remove the need for a semicolon in a separate PR

Reply:
> Hah, [apparently not](https://standardjs.com/#i-disagree-with-rule-x-can-you-change-it)

Ha, yeah and I think we already chose Standard over ESLint (or semistantard) as an organisation

Reply:
> Love the pro tip! This semicolon is actually required because of the iife and passes the checks. An option would be to rewrite this as a module

Oh that's interesting. So it's different to the line before, even though they require the same thing?

I can't remember which is the more modern syntax, but is this better than `import elasticlunr as lunr from elasticlunr` or `import { elasticlunr }...`?

Reply:
You're right `import` is the more modern syntax. I have updated this which seems to have negated the need for the semi colon.

Elasticlunr is the default export, and I cannot import lunr for the use of `lunr.stemmer` - but there seems to be a stemmer function available on elasticlunr. Not sure on the context of why lunr was used originally here, but I have tested search and it seems to be working fine, but would be great if either of you can give search a quick test locally  to check.

Reply:
Have tested locally too and all seems to be as expected

Reply:
> Ha, yeah and I think we already chose Standard over ESLint (or semistantard) as an organisation
From memory the reason we settled on Standard was something like 'we use it for Ruby so lets give it a go for JS'. If we're finding it's not consistently linting in a way that works for us I suggest we use something else EG semistandard